watertap/unit_models/zero_order/tests/test_surface_discharge.py

```diff
@@ -0,0 +1,330 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+"""
+Tests for zero-order surface discharge model
+"""
+import pytest
+
+from io import StringIO
+from pyomo.environ import (
+    ConcreteModel, Constraint, Param, Block, value, Var, assert_optimal_termination)
+from pyomo.util.check_units import assert_units_consistent
+
+from idaes.core import FlowsheetBlock
+from idaes.core.util import get_solver
+from idaes.core.util.model_statistics import degrees_of_freedom
+from idaes.core.util.testing import initialization_tester
+from idaes.generic_models.costing import UnitModelCostingBlock
+
+from watertap.unit_models.zero_order import SurfaceDischargeZO
+from watertap.core.wt_database import Database
+from watertap.core.zero_order_properties import WaterParameterBlock
+from watertap.core.zero_order_costing import ZeroOrderCosting
+
+solver = get_solver()
+
+class TestSurfaceDischargeZO:
+    @pytest.fixture(scope="class")
+    def model(self):
+        m = ConcreteModel()
+        m.db = Database()
+
+        m.fs = FlowsheetBlock(default={"dynamic": False})
+        m.fs.params = WaterParameterBlock(
+                default={"solute_list": ["toc", "nitrate", "sulfate", "bar", "crux"]})
```

Comment:
What is `crux` in this case, and why is it needed?

Reply:
You already have `"bar"` for a dummy component in this case however.

Reply:
True. I will remove "crux" on my next update.

Reply:
I addressed this on my most recent update, so I am marking resolved here.